### DISCUSSION TITLE
How can we create cubic pore network from image.array

### DISCUSSION THREAD

**Discussion by pengzhulin** [PMEAL/OpenPNM#2887] [Q&A] [answered]
Thank you for your tutorial of **Import an Extracted Network and Predict Transport Properties**. Snow2 can provide a simplified but irregular instead of cubic pore network. And I notcie that in Porespy, there is use "net = op.network.CubicTemplate(template=im, spacing=1.0)" to create cubic pore network. But the use of CubicTemplate may generate too big network. Is there other method to create regular (cubic) pore network from image arrays? Thank you!

**Comment by jgostick** [comment, marked as answer]
Not really, you can just downsize your image, using something like `scipy.ndimage.zoom`.
